package.json

```diff
@@ -18,12 +18,11 @@
     "styleguide"
   ],
   "peerDependencies": {
-    "babel-eslint": "^8.0.1",
     "eslint": "^4.9.0",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-react": "^7.4.0"
+    "eslint-config-skyscanner": "^3.0.0",
```

Comment:
Keeping these as peer dependencies means either a change to `backpack-react-scripts` (to add them as dependencies there, see https://github.com/Skyscanner/backpack-react-scripts/blob/96c5c95bd571c96f693ea950a5adcbb726d2672b/packages/react-scripts/package.json#L35-L39) or a requirement for the user to install these deps themselves in the top-level project.